tests/phpunit/tests/rest-api/rest-plugins-controller.php

```diff
@@ -1017,7 +1017,12 @@ protected function check_get_plugin_data( $data, $network_only = false ) {
 	}
 
 	/**
-	 * Sets up the plugin download to come locally instead of downloading it from .org
+	 * Sets up the plugin repository requests to use local data.
+	 *
+	 * Request to the plugin repository are mocked to avoid external HTTP requests so
+	 * the test suite does not produce false negatives due to network failures.
+	 *
+	 * Both the plugin zip file and the plugin API response are mocked.
```

Comment:
```suggestion
	 * Requests to the plugin repository are mocked to avoid external HTTP requests so
	 * the test suite does not produce false negatives due to network failures.
	 *
	 * Both the plugin ZIP file and the plugin API response are mocked.
```
I will hunt you down if you keep typing `zip` instead of `ZIP`.